pkg/cli/tsdump_upload.go

```diff
@@ -226,7 +227,7 @@ func (d *datadogWriter) emitDataDogMetrics(data []DatadogSeries) error {
 	}
 
 	fmt.Printf(
-		"tsdump datadog upload: sending payload containing %d series including %s\n",
+		"\033[G\033[K tsdump datadog upload: uploading metrics containing %d series including %s",
```

Comment:
Have you tested this with retry errors?
If there are more line appended to STDOUT, the output might get messed up because of the miscalculated `\033[G\033[K`